src/elements/select/select.ssr.spec.ts

```diff
@@ -9,20 +9,48 @@ import '../option.js';
 describe(`sbb-select ssr`, () => {
   let root: SbbSelectElement;
 
-  beforeEach(async () => {
+  it('renders', async () => {
     root = await ssrHydratedFixture(
       html`
-        <sbb-select placeholder="Placeholder">
+        <sbb-select placeholder="Placeholder" value="1">
```

Comment:
Should this be reverted (test with no value)?

Reply:
Thanks! Will fix this